includes/admin/class-wcs-admin-post-types.php

```diff
@@ -244,57 +244,69 @@ public function remove_bulk_actions( $actions ) {
 	}
 
 	/**
-	 * Add extra options to the bulk actions dropdown
-	 *
-	 * It's only on the All Shop Subscriptions screen.
-	 * Introducing new filter: woocommerce_subscription_bulk_actions. This has to be done through jQuery as the
-	 * 'bulk_actions' filter that WordPress has can only be used to remove bulk actions, not to add them.
-	 *
-	 * This is a filterable array where the key is the action (will become query arg), and the value is a translatable
-	 * string. The same array is used to
+	 * Remove "edit" from the bulk actions.
 	 *
+	 * @param array $actions
+	 * @return array
 	 */
-	public function print_bulk_actions_script() {
+	public function filter_bulk_actions( $actions ) {
+		// 'post_status' is used in CPT datastore, 'status' is used in HPOS datastore.
+        // phpcs:ignore WordPress.Security.NonceVerification.Recommended,WordPress.Security.ValidatedSanitizedInput.MissingUnslash,WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		$post_status = $_GET['post_status'] ?? $_GET['status'] ?? '';
+
+		// List of actions to remove that are irrelevant for subscriptions
```

Comment:
```suggestion
		// List of actions to remove that are irrelevant for subscriptions.
```

Nit: fullstop at end of comment.

Reply:
addressed in 225e2ad3